kuma/users/tests/test_views.py

```diff
@@ -1209,7 +1177,7 @@ def test_signup_public_github(self, is_public=True):
         data = {
             "website": "",
             "username": "octocat",
-            "email": "octo.cat@github-inc.com",
+            "email": "octocat-private@example.com",
```

Comment:
I know this test wasn't related but it became. Now that you're not allowed to POST in an email that isn't one of your OAuth emails (from the scope), all the tests that didn't adhere needed to change too.